src/models/User.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db.js');
+
+const User = sequelize.define('user', {
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
```

Comment:
The 'email' field is unique but there is no provided strategy for handling cases where a duplicate email is entered. Implement a strategy to handle such cases.

Reply:
it is done on the application level